controllers/ibmblockcsi_controller.go

```diff
@@ -64,18 +61,19 @@ var daemonSetRestartedValue = ""
 
 var log = logf.Log.WithName("ibmblockcsi_controller")
 
-type reconciler func(instance *ibmblockcsi.IBMBlockCSI) error
+type reconciler func(instance *controller_instance.IBMBlockCSI) error
 
 // IBMBlockCSIReconciler reconciles a IBMBlockCSI object
 type IBMBlockCSIReconciler struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
 	client.Client
-	Log           logr.Logger
-	Scheme        *runtime.Scheme
-	Namespace     string
-	Recorder      record.EventRecorder
-	ServerVersion string
+	Log              logr.Logger
+	Scheme           *runtime.Scheme
+	Namespace        string
+	Recorder         record.EventRecorder
+	ServerVersion    string
+	ControllerHelper *common.ControllerHelper
```

Comment:
please consider adding it to the [initialization in main](https://github.com/IBM/ibm-block-csi-operator/blob/060bde032636e530a43764b00802cb03f5ec0870/main.go#L90)

Reply:
done